Client/package.json

```diff
@@ -32,7 +32,8 @@
     "react-router-dom": "^6.23.1",
     "react-toastify": "^10.0.5",
     "recharts": "2.12.7",
-    "redux-persist": "6.0.0"
+    "redux-persist": "6.0.0",
+    "vite-plugin-svgr": "^4.2.0"
```

Comment:
Is there a signifcant benefit to usign this plugin rather than an `img` with the svg as the source?

I'm always of the opinion that the fewer dependencies a project has the better, but if there is a real benefit then disregard

Reply:
I tend to agree with you but `vite-plugin-svgr` is a super lightweight tool. Reason why I added it is for ease of maintenance in case we ever do decide to mess around with the svgs or import more non @mui icons. Makes it easier to edit within the code, rather than having to locate the files and update them separately. I believe svgs also offer better performance, although I could be wrong.